go-controller/pkg/cni/ovs.go

```diff
@@ -66,3 +68,30 @@ func ovsClear(table, record string, columns ...string) error {
 	_, err := ovsExec(args...)
 	return err
 }
+
+func ofctlExec(args ...string) (string, error) {
+	args = append([]string{"--timeout=30"}, args...)
+	output, err := runner.Command("ovs-ofctl", args...).CombinedOutput()
+	if err != nil {
+		return "", fmt.Errorf("failed to run 'ovs-ofctl %s': %v\n  %q", strings.Join(args, " "), err, string(output))
+	}
+
+	outStr := string(output)
+	trimmed := strings.TrimSpace(outStr)
+	// If output is a single line, strip the trailing newline
+	if strings.Count(trimmed, "\n") == 0 {
+		outStr = trimmed
+	}
+
+	return outStr, nil
+}
+
+func waitForBrIntFlows(ip string) error {
```

Comment:
nit: this should be renamed to something more specific, `waitForIPFlowsOnBridge` or something like that.

Reply:
Or if this is going to be a generic mechanism to check for flows existing then call it `waitForFlowsOnBridge(retry, timeout time.Duration, flow string)`

Reply:
It's not supposed to be generic. In dcbw's original downstream hack `waitForBrIntFlows` was internal to helper_linux.go and it should probably be done that way here too.

Reply:
I renamed it to waitForPodFlows() which is what it's literally doing.